workflows/nallo.nf

```diff
@@ -497,7 +497,7 @@ workflow NALLO {
     softwareVersionsToYAML(ch_versions)
         .collectFile(
             storeDir: "${params.outdir}/pipeline_info",
-            name: 'nf_core_pipeline_software_mqc_versions.yml',
+            name:  ''  + 'pipeline_software_' +  'mqc_'  + 'versions.yml',
```

Comment:
Why the first `''`?

Reply:
Guess `''` holds `nf_core_` in nf-core pipelines.

Reply:
I'll change it to `nallo_`.